runtimes/pythonrt/py-sdk/autokitteh/github.py

```diff
@@ -47,7 +47,13 @@ def github_client(connection: str, **kwargs) -> Github:
     if not install_id:
         raise ConnectionInitError(connection)
 
-    app = GithubIntegration(auth=AppAuth(int(app_id), connection), **kwargs)
+    app = None
+    private_key = os.getenv(f"{connection}__private_key")
+    if private_key:
+        # Exposing the private key is fine here as it's a custom OAuth and stored in connection variables.
```

Comment:
The reader of this code doesn't need to know about AK's concept of "custom" OAuth, or connection vars, just why it's not a security risk.

```suggestion
        # Exposing the private key is fine here as it belongs to the user.
```